app/javascript/components/enrollment/Enrollment.js

```diff
@@ -180,22 +183,19 @@ class Enrollment extends React.Component {
     let index = this.state.index;
     let lastIndex = this.state.lastIndex;
     if (lastIndex) {
-      this.setState({ direction: 'next' }, () => {
-        this.setState({ index: lastIndex, lastIndex: null });
-      });
+      this.setState({ index: lastIndex, lastIndex: null });
     } else {
-      this.setState({ direction: 'next' }, () => {
-        this.setState({ index: index + 1, lastIndex: null });
-      });
+      this.setState({ index: index + ((this.state.enrollmentState.isolation ? 4 : 5) == index ? 2 : 1), lastIndex: null });
```

Comment:
=== instead of ==

Reply:
fixed in https://github.com/SaraAlert/SaraAlert/pull/975/commits/441588e6694ae61c4bd1da5b7da3be4de8fd5aa9